stdlib/bytes.ml

```diff
@@ -418,7 +420,7 @@ let of_seq i =
   let buf = ref (make 256 '\000') in
   let resize () =
     (* resize *)
-    let new_len = min (2 * length !buf) Sys.max_string_length in
+    let new_len = int_min (2 * length !buf) Sys.max_string_length in
```

Comment:
I wonder if it is serves any purpose to duplicate the definition of `Int.min` to avoid depending on `Int`. As you mentioned in some other place, `Int` is tiny and depends itself only on `Stdlib`, so it seems pretty harmless to add that dependency to this module (and other similar modules).